src/cursor/abstract_cursor.ts

```diff
@@ -193,6 +193,10 @@ export abstract class AbstractCursor<
     return this[kSession];
   }
 
+  set session(clientSession: ClientSession | undefined) {
```

Comment:
Setter added as a convenience for the unified test runner to set named sessions on operations that return a cursor.

Reply:
We should keep this private you can find a helper in test/tools/utils.js called `getSymbolFrom` that will help with grabbing `kSession`.